components/empty-state/getIllustration.js

```diff
@@ -0,0 +1,25 @@
+import { html } from 'lit';
+import { loadSvg } from '../../generated/empty-state/presetIllustrationLoader.js';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+
+function fixSvg(svgStr) {
+
+	if (svgStr === undefined) {
+		return undefined;
+	}
+
+	const template = document.createElement('template');
+	template.innerHTML = svgStr;
+
+	return html`${unsafeSVG(template.innerHTML)}`;
+
+}
+
+export async function getIllustration(illustration) {
+
+	if (illustration) {
+		const svg = await loadSvg(illustration);
+		return fixSvg(svg ? svg.val : undefined);
+	}
+
+}
```

Comment:
Nit on the illustration filenames (I can't comment on the SVGs!): I'd go with `checklist.svg` and `desert-road.svg` since "illustration" is redundant and we typically go with hyphens not underscores to separate words in filenames.

Reply:
Yep, these were more of a placeholder for my testing. We still need to have a chat about the illustrations and illustration names we will be using.